space-x-app/Views/RocketCollectionViewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  RocketCollectionViewCell.swift
+//  space-x-app
+//
+//  Created by Valerii Klymenko on 27/03/2020.
+//  Copyright © 2020 Valerii Klymenko. All rights reserved.
+//
+
+import UIKit
+
+class RocketCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak var rocketName: UILabel!
+    @IBOutlet weak var cellImage: UIImageView!
+    
+    lazy private(set) var viewModel = RocketsImageViewModel(apiClient: ApiClient.shared, collectionViewCell: self)
```

Comment:
why not move this to the configure method? You dont need the viewModel outside it anyways